### PR TITLE
Update master to main 

### PR DESCRIPTION
**Pull request by alxogm**
in requirements.txt and github/workflows/python-package.yml

### CODE REVIEW

**Comment by alxogm** [comment]
Tests also fails here, but due to something related to a library in simqso, see error below. Should I merge anyway @sbailey ?
 
```

Collecting simqso
[237](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:238)
  Cloning https://github.com/desihub/simqso.git (to revision main) to /tmp/pip-install-pbx3ckk_/simqso_319c6291fef44b1985171fc17dee468d
[238](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:239)
  Running command git clone --filter=blob:none --quiet https://github.com/desihub/simqso.git /tmp/pip-install-pbx3ckk_/simqso_319c6291fef44b1985171fc17dee468d
[239](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:240)
  Resolved https://github.com/desihub/simqso.git to commit c3e416e62dccb9205f2bd3911f4e7fb38538cd38
[240](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:241)
  Preparing metadata (setup.py): started
[241](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:242)
  Preparing metadata (setup.py): finished with status 'error'
[242](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:243)
  error: subprocess-exited-with-error
[243](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:244)
  
[244](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:245)
  × python setup.py egg_info did not run successfully.
[245](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:246)
  │ exit code: 1
[246](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:247)
  ╰─> [24 lines of output]
[247](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:248)
      Traceback (most recent call last):
[248](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:249)
        File "<string>", line 2, in <module>
[249](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:250)
        File "<pip-setuptools-caller>", line 34, in <module>
[250](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:251)
        File "/tmp/pip-install-pbx3ckk_/simqso_319c6291fef44b1985171fc17dee468d/setup.py", line 10, in <module>
[251](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:252)
          from simqso import __version__
[252](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:253)
        File "/tmp/pip-install-pbx3ckk_/simqso_319c6291fef44b1985171fc17dee468d/simqso/__init__.py", line 5, in <module>
[253](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:254)
          from .sqrun import qsoSimulation
[254](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:255)
        File "/tmp/pip-install-pbx3ckk_/simqso_319c6291fef44b1985171fc17dee468d/simqso/sqrun.py", line 7, in <module>
[255](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:256)
          from astropy.table import Table,hstack
[256](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:257)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/table/__init__.py", line 4, in <module>
[257](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:258)
          from .column import Column, MaskedColumn, StringTruncateWarning, ColumnInfo
[258](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:259)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/table/column.py", line 12, in <module>
[259](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:260)
          from astropy.units import Unit, Quantity
[260](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:261)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/units/__init__.py", line 17, in <module>
[261](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:262)
          from .quantity import *
[262](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:263)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/units/quantity.py", line 32, in <module>
[263](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:264)
          from .quantity_helper import (converters_and_unit, can_have_arbitrary_unit,
[264](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:265)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/units/quantity_helper/__init__.py", line 10, in <module>
[265](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:266)
          from . import helpers, function_helpers
[266](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:267)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/astropy/units/quantity_helper/function_helpers.py", line 116, in <module>
[267](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:268)
          np.asscalar, np.alen,
[268](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:269)
        File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/numpy/__init__.py", line 311, in __getattr__
[269](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:270)
          raise AttributeError("module {!r} has no attribute "
[270](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:271)
      AttributeError: module 'numpy' has no attribute 'asscalar'
[271](https://github.com/desihub/desisim/actions/runs/3315173983/jobs/5475415168#step:4:272)
      [end of output]
```

**Comment by alxogm** [comment]
Yeii, now all tests pass. I'll just wait for approval in order to merge ;D

**Review by sbailey — APPROVED** [review]
Looks good, thanks.  I'm glad you found the numpy version fix for the asscalar problem too.

**Pull request merged**
